scripts/setup-operator-sdk.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+version="v0.10.0"
+
+mkdir -p $GOPATH/src/github.com/operator-framework
+cd $GOPATH/src/github.com/operator-framework
+git clone --branch ${version} --depth 1 https://github.com/operator-framework/operator-sdk
+cd operator-sdk
```

Comment:
Why not just do `go get`?

Reply:
I found no way to get a specified version with `go get`. It seems the official solution is "use modules"...